src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -349,6 +350,11 @@ public DevUtil(File serverDirectory, File sourceDirectory, File testSourceDirect
         if (projectDirectory != null) {
             this.defaultDockerfile = new File(projectDirectory, "Dockerfile");
         }
+        if (dockerTimeout < 1 || dockerTimeout > 3600) {
```

Comment:
It may be simpler (less confusing in the documentation) if we just don't have an upper bound.  i.e. easier to understand that it's in seconds and must be greater than zero.

Reply:
I'm just afraid they'll mistakenly put in a really big number and then docker will take a long time and then they'll complain dev mode is hanging. But that is a pathological failure so you could say an edge case. I'll remove the top limit unless you're worried about the same thing.

Reply:
That is possible but I agree it's an edge case.  Even having 3600 as a limit might be misleading because users should really use a lower number and do something else to speed up the docker build (e.g. remove unnecessary things from the build context), rather than wait an hour.